llvmlibc-samples/src/llvmlibc/baremetal-semihosting/microbit-llvmlibc.ld

```diff
@@ -0,0 +1,240 @@
+/*
+ * SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright © 2019 Keith Packard
```

Comment:
Do we need to add Arm copyright as well?

Reply:
It was a trivial modification, but it is probably best to add the additional copyright anyway.